speechmatics_flow/models.py

```diff
@@ -111,20 +115,39 @@ class ServerMessageType(str, Enum):
     AddTranscript = "AddTranscript"
     """Indicates the final transcript of a part of the audio."""
 
+    ResponseStarted = "ResponseStarted"
+    """
+    Indicates the start of TTS audio streaming from the server.
+    The message contains the textual content of the utterance to be spoken.
+    """
+
+    ResponseCompleted = "ResponseCompleted"
+    """Indicates the completion of TTS audio transmission from the server.
+    The message includes the textual content of the utterance just spoken.
+    """
+
+    ResponseInterrupted = "ResponseInterrupted"
+    """Indicates an interruption in the TTS audio stream from the server.
+    The message contains the textual content up to the point where the utterance was stopped.
+    """
+
+    AddAudio = "AddAudio"
+    """Implicit name for all outbound binary messages. The client confirms
+    receipt by sending an :py:attr:`ServerMessageType.AudioReceived` message."""
+
     audio = "audio"
     """Message contains binary data"""
 
     prompt = "prompt"
```

Comment:
Should there be a message to mark this will be deprecated and link to docs https://docs.speechmatics.com/flow/flow-api-ref#prompt-deprecated ?